src/styles/main.scss

```diff
@@ -0,0 +1,57 @@
+$width-box: 100px;
+$gap: 1px;
+$days-of-week: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
+
+  &__day {
+    box-sizing: border-box;
+    width: $width-box;
+    height: $width-box;
+    background-color: #eee;
+    border: 1px solid #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition-duration: 0.5s;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: #ffbfcb;
+      cursor: pointer;
+      transition-duration: 0.5s;
```

Comment:
The `transition-duration` property is repeated in the `&:hover` block. It's already defined in line 30, so you can remove it here to avoid redundancy.